src/guide/jbs-jdk-bug-system.md

```diff
@@ -214,6 +214,7 @@ When triaging an issue, first give it a general review.
 
 1. If the issue is a duplicate, close it as such.
 1. If the issue belongs to a different area (it was filed in libraries, but it's an HotSpot issue), transfer it to the correct component/subcomponent making sure that the state remains [New]{.jbs-value}.
+   * Please note that issues in the [hotspot]{.jbs-value} and [security-libs]{.jbs-value} components must have a [Subcomponent]{.jbs-field} set as well.
```

Comment:
@irisclark is right, it's advisable to set a subcomponent for issues in client-libs. However, it's not required as far as I can tell. There are issues where subcomponent isn't set, in majority of cases this is likely because the issue is too broad and is applicable to the entire client area.